src/python/strelka/tests/test_scan_tlsh.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from unittest import TestCase, mock
+
+from strelka.scanners.scan_tlsh import ScanTlsh as ScanUnderTest
+from strelka.tests import run_test_scan
+
+
+def test_scan_tlsh(mocker):
+    """
+    Pass: Sample event matches output of scanner.
+    Failure: Unable to load file or sample event fails to match.
+    """
+    test_scan_event = {
+        "elapsed": mock.ANY,
+        "match": {"family": "TestMatchA", "score": 0},
+        "flags": [],
+    }
+
+    scanner_event = run_test_scan(
+        mocker=mocker,
+        scan_class=ScanUnderTest,
+        fixture_path=Path(__file__).parent / "fixtures/test.txt",
+        options={"location": str(Path(Path(__file__).parent / "fixtures/test_tlsh.yaml"))}
```

Comment:
It's not documented or intuitive, but any file that matches `fixtures/test_*` glob will get fed into test_scanner_assignment. This will cause the test to fail. I move files that aren't designed to be fed into Strelka (e.g. test_passwords.dat) into the `helpers/` directory to keep them out of the scanner assignment test. I can think through how to make that more obvious before adding `--build-arg CONFIG_TESTS=true` to the Github Actions.